package/cpp/rnskia/RNSkDrawView.cpp

```diff
@@ -49,12 +49,69 @@ RNSkDrawView::~RNSkDrawView() {
   endDrawingLoop();
 }
 
+void RNSkDrawView::setJsiProperties(std::unordered_map<std::string, JsiValueWrapper> &props) {
+  for(auto& prop: props) {
+    if(prop.first == "drawCallback") {
+      if(prop.second.isUndefinedOrNull()) {
+        // Clear drawcallback
+        _drawCallback = nullptr;
+        // We can just reset everything - this is a signal that we're done.
+        endDrawingLoop();
+        return;
+      } else if (prop.second.getType() != JsiWrapperValueType::Function) {
+        // We expect a function for the draw callback custom property
+        throw std::runtime_error("Expected a function for the drawCallback custom property.");
+      }
+
+      // Save callback
+      _drawCallback = prop.second.getAsFunction();
+
+      // Request redraw
+      requestRedraw();
+
+    } else {
+      throw std::runtime_error("Property " + prop.first + " not found.");
+    }
+  }
+}
+
+jsi::Value RNSkDrawView::callJsiMethod(jsi::Runtime& runtime,
+                                       const std::string& name,
+                                       const jsi::Value *arguments,
+                                       size_t count) {
+  
+  if (name == "invalidate") {
```

Comment:
`redraw()`dispatches `invalidate()` that dispatches `requestRedraw()`, should `invalidate` be renamed?